x-pack/legacy/plugins/siem/cypress/urls/navigation.ts

```diff
@@ -6,3 +6,10 @@
 
 export const TIMELINES_PAGE = '/app/siem#/timelines';
 export const OVERVIEW_PAGE = '/app/siem#/overview';
+export const HOSTS_PAGE_TAB_URLS = {
+  allHosts: '/app/siem#/hosts/allHosts',
+  anomalies: '/app/siem#/hosts/anomalies',
+  authentications: '/app/siem#/hosts/authentications',
+  events: '/app/siem#/hosts/events',
+  uncommonProcesses: '/app/siem#/hosts/uncommonProcesses',
```

Comment:
Shall we add a link for `external alerts` as well?

Reply:
I'll take a look at that, thx!! ^^

Reply:
I'm fine if have another PR to handle that as well!